styles/connectionsetup.css

```diff
@@ -34,3 +34,15 @@ ul {
 #connectionStatus {
     display: inline-block;
 }
+
+#tokenField {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 1rem;
```

Comment:
FYI, I moved this and the duplicate above to a CSS class on my branch, I'll take care of it when I'll rebase.